asaplib/kernel/ml_kernel_operations.py

```diff
@@ -8,7 +8,19 @@
 
 
 def normalizekernel(kernel):
-    # first normalize the kernel matrix
+    """
```

Comment:
This is the module I wrote the tests for. Just expanding the docstrings slightly.